NEWS.md

```diff
@@ -1,3 +1,7 @@
+MixedModels v4.9.1 Release Notes
+==============================
+* Revise code in benchmarks to work with recent Julia and PkgBenchmark.jl [#667]
+
 MixedModels v4.9.0 Release Notes
 ==============================
 * Julia minimum compat version raised to 1.8 because of BSplineKit [#665]
```

Comment:
```suggestion
MixedModels v4.9.0 Release Notes
==============================
* Revise code in benchmarks to work with recent Julia and PkgBenchmark.jl [#667]
* Julia minimum compat version raised to 1.8 because of BSplineKit [#665]
```

we haven't tagged 4.9 yet :smile: